blog/service/core/src/main/kotlin/com/excref/kotblog/blog/service/post/domain/Post.kt

```diff
@@ -0,0 +1,33 @@
+package com.excref.kotblog.blog.service.post.domain
+
+import com.excref.kotblog.blog.service.blog.domain.Blog
+import com.excref.kotblog.blog.service.category.domain.Category
+import com.excref.kotblog.blog.service.common.UuidAwareDomain
+import com.excref.kotblog.blog.service.tag.domain.Tag
+import javax.persistence.*
+
+/**
+ * @author Gugo dzya
+ * @since 6/10/17 6:59 PM
+ */
+@Entity
+class Post(
+        @Column(name = "name", nullable = false)
+        val name: String,
```

Comment:
I thought it is a good place to hold short title.. might be useful for preview, and search priorities in future.